usvg/src/svgtree/names.rs

```diff
@@ -136,8 +136,8 @@ impl EId {
     }
 
     #[inline(never)]
-    pub fn to_str(&self) -> &'static str {
-        ELEMENTS.key(self)
+    pub fn to_str(self) -> &'static str {
```

Comment:
You have to update codegen too then: https://github.com/RazrFalcon/resvg/blob/11f438346386f02c5f004706bfa712f8f93c5810/usvg/codegen/main.rs#L142